codeworld-base/src/Internal/Num.hs

```diff
@@ -77,7 +77,7 @@ module Internal.Num (
 
 import qualified "base" Prelude as P
 import "base" Prelude (Bool(..), (.), (==), map)
-import Numeric
+import Numeric hiding (sqrt, pi, exp, log, logBase, sin, tan, cos, asin, atan, acos)
```

Comment:
Actually, I'll merge this and then change it to an explicit import of `showFFloatAlt`.  That is backward compatible, and is all that was really needed anyway.